### PR TITLE
infra!: adhere to monorepo strucutre of open-scd

### LINKED ISSUE #282: Make CoMPAS-OpenSCD adhere to the monorepository structure
In order to move forward, we need to make CoMPAS OpenSCD adhere to the monorepository structure that has been implemented by OpenSCD.

What we need to do for this, is to move all the related files into `packages/compas-openscd`.

Move the following:
- [ ] public
- [ ] src
- [ ] test
- [ ] .editorconfig
- [ ] .eslintrc.cjs
- [ ] .gitignore
- [ ] .prettierignore
- [ ] CHANGELOG.md
- [ ] Dockerfile
- [ ] ModHeader.png
- [ ] README.md
- [ ] RELEASING.md
- [ ] browserconfig.xml
- [ ] favicon.ico
- [ ] index.html
- [ ] manifest.json
- [ ] package-lock.json
- [ ] package.json
- [ ] snowpack.config.mjs
- [ ] sw.js
- [ ] tsconfig.json
- [ ] web-test-runner.config.mjs
- [ ] workbox-config.cjs

After moving these files, we need to update the corresponding GitHub Action files:

2) Update .github folder

A) Update .github/dependabot.yml
The package-ecosystem `docker` should have directory `packages/compas-openscd/`, instead of `/`

B) Update .github/build-project.yml
Update the steps `Test application with npm`, `Build application with npm`, `Build docker image`: `cd packages/compas-open-scd && ..`

C) Update .github/release-project.yml
Update the steps `Build application with npm`, `Build and push docker image to Docker Hub`: `cd packages/compas-open-scd && ..`

D) Update .github/test-and-build.yml
Update the step `Install and Build`: `cd packages/compas-open-scd && ..`

E) Update .github/test.yml
Update the step `Install and Test`: `cd packages/compas-open-scd && ..`

When this is done, we need to add a couple of new files in the Repository root:
- package.json
```json
{
  "name": "compas-openscd-monorepo",
  "version": "0.0.1",
  "description": "CoMPAS OpenSCD base distribution and plugins",
  "private": true,
  "workspaces": [
    "packages/*"
  ],
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/com-pas/compas-open-scd.git"
  },
  "license": "Apache-2.0"
}
```

- .gitignore (Can be a copy of packages/compas-openscd)

- README.md, write something fancy here. Example is in https://github.com/openscd/open-scd.
Link to the README from compas-openscd (packages/compas-openscd/README.md)

Acceptance Criteria:

- [ ] GitHub Actions still work.
- [ ] Building, deploying and publishing still works.
- [ ] Markdown files are correctly linked

### PR DESCRIPTION
**Pull request by juancho0202**
Closes #282

### CODE REVIEW

File: packages/compas-open-scd/docs/decisions/0001-record-architecture-decisions.md
Comment (Stef3st) [inline packages/compas-open-scd/docs/decisions/0001-record-architecture-decisions.md:1]: This should be moved out of the compas-open-scd package

**Review by Stef3st — CHANGES_REQUESTED** [review]
Looks good to me. Id suggest to move the document folder and add a Apache-2.0 license on repository level

File: packages/compas-open-scd/docs/decisions/0001-record-architecture-decisions.md
Comment (juancho0202) [reply packages/compas-open-scd/docs/decisions/0001-record-architecture-decisions.md:1]: both done ✔️

**Review by Stef3st — DISMISSED** [review]
LGTM!

**Pull request merged**